src/monexamine.cpp

```diff
@@ -427,6 +427,10 @@ void monexamine::swap( monster &z )
             z.add_effect( effect_tied, 1_turns, num_bp, true );
         }
         add_msg( _( "You swap positions with your %s." ), pet_name );
+
```

Comment:
This is going to be unnecessary after #35556 is merged, because monexamine::swap() will call `game::swap_critters()`.

Reply:
And now #35556 is merged, so this is unnecessary.